app.py

```diff
@@ -32,18 +30,27 @@ def __init__(self):
         else:
             self.drv = None
             self.enable_pin = None
+
         eventbus.on(ButtonDownEvent, self._handle_buttondown, self)
-        eventbus.on_async(HexpansionInsertionEvent, self.handle_hexpansion_insertion, self)
-        eventbus.on_async(HexpansionRemovalEvent, self.handle_hexpansion_removal, self)
+        eventbus.on(HexpansionInsertionEvent, self.handle_hexpansion_insertion, self)
+        eventbus.on(HexpansionRemovalEvent, self.handle_hexpansion_removal, self)
 
 
-    def handle_hexpansion_insertion(self):
+    def handle_hexpansion_insertion(self, event):
         self.hexpansion_config = self.scan_for_hexpansion()
+        self.last_jitter_time = time.ticks_ms()
```

Comment:
If you remove and insert the hexpansion, you need to reset these, especially cuz people could be inserting the hexpansions in a nother slot.